src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -129,9 +150,8 @@ fun startConsumer(
     applicationState: ApplicationState,
     topic: String,
     kafkaConsumerPapirSmRegistering: KafkaConsumer<String, String>,
-    database: Database,
-    oppgaveClient: OppgaveClient,
-    source: String
+    receivedSykmeldingController: ReceivedSykmeldingController,
+    source: String,
```

Comment:
source kan nok tas bort, det var noe vi la til for å se om appen leste fra aiven eller onprem ifm bytting :)

Reply:
Ah, smart!